bin/projects/dbatools/dbatools/Parameter/DbaInstanceParameter.cs

```diff
@@ -400,7 +400,8 @@ public DbaInstanceParameter(object Input)
                 case "microsoft.sqlserver.management.smo.server":
                     try
                     {
-                        _ComputerName = (string)tempInput.Properties["NetName"].Value;
```

Comment:
Can we do a keyExists() check instead of a try catch? throwing an exception is expensive (it locks every core, granted for nanoseconds). exceptions should be for stuff we really don't expect to happen.